paasta_tools/kubernetes_tools.py

```diff
@@ -642,6 +535,7 @@ def get_autoscaling_metric_spec(
             if (
                 autoscaling_params.get("forecast_policy") == "moving_average"
                 or "offset" in autoscaling_params
+                or load_system_paasta_config().get_hpa_always_uses_external_for_signalfx()
             ):
                 hpa_metric_name = self.namespace_external_metric_name(metrics_provider)
                 signalflow = LEGACY_AUTOSCALING_SIGNALFLOW.format(
```

Comment:
I think this means that all instances using the `metrics_provider: uwsgi` or `http` will end up with an implicit `forecast_policy: moving_average` -- is that what we want?

Reply:
(oops meant to start a review -- not done!)

Reply:
(There probably aren't too many instances using `uwsgi` or `http` without `moving_average` I bet)